scripts/baseline_comparison/evaluate_utils.py

```diff
@@ -27,11 +27,15 @@
 class Experiment:
     expname: str  # name of the parent experiment used to store the file
     name: str  # name of the specific experiment, e.g. "localsearch"
-    run_fun: Callable[[], List[ResultRow]]  # function to execute to obtain results
+    run_fun: Callable[..., List[ResultRow]]  # function to execute to obtain results
+    kwargs: dict = None
 
-    def data(self, ignore_cache: bool = False):
+    def data(self, ignore_cache: bool = False) -> pd.DataFrame:
```

Comment:
Oh I had forgotten this code 🙈

Reply:
The old version had some pretty crazy weirdness going on due to what I think was mutable lambda functions when the lambda's are taking in variables that are edited after the lambda's creation, causing them to use the new values rather than the old intended values. By instead passing the kwargs at the time of calling `.data()`, this avoids this problem.